### PR TITLE
Reframe our positioning

### PR DESCRIPTION
**Pull request by ramonguiu**
## Description
Reframe our positioning and provide clearer more actionable steps on how to get started with Promscale for our two main users

## Merge requirements

Please take into account the following non-code changes that you may need to make with your PR:

- [ ] CHANGELOG entry for user-facing changes
- [ ] Updated the relevant documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,61 +13,118 @@
...
+To get started:
+1. [Install Promscale](https://docs.timescale.com/promscale/latest/installation/#install-promscale-with-instrumentation).
+2. [Configure Prometheus](https://docs.timescale.com/promscale/latest/send-data/prometheus/) to send data to Promscale.
+3. [Configure Grafana](https://docs.timescale.com/promscale/latest/visualize-data/grafana/) to query and visualize metrics from Promscale
+using a PromQL and/or a PostgreSQL datasource.
+
+## Promscale for Jaeger and OpenTelemetry
+
+Promscale can natively ingest OpenTelemetry traces and it can also ingest Jaeger and Zipkin traces via
+the OpenTelemetry Collector.
```
Comment (Harkishen-Singh) [inline README.md:64]:
```suggestion
Promscale supports ingesting OpenTelemetry traces natively, and Jaeger and Zipkin traces via
the OpenTelemetry Collector.
```
WDYT?

File: README.md [hunk 2]
```diff
@@ -13,61 +13,118 @@
...
+
+Promscale solves three main use cases for Jaeger and OpenTelemetry users:
+
+* **Easy-to-use durable and scalable storage backend for traces**<br/>
```
Comment (Harkishen-Singh) [inline README.md:68]:
```suggestion
* **Easy-to-use durable and scalable storage**<br/>
```

File: README.md [hunk 3]
```diff
@@ -13,61 +13,118 @@
...
+
+Promscale is a unified metric and trace observability backend for Prometheus,
+Jaeger and OpenTelemetry built on PostgreSQL and TimescaleDB.
+
+Promscale serves as a robust and 100% PromQL-compliant Prometheus remote storage
+and as a durable and scalable Jaeger storage backend.
+
+Unlike other observability backends, it has a simple
+and easy-to-manage architecture with just two components: the Promscale
+Connector and the Promscale Database.
```
[the remark this answers is not in the document]
Comment (ramonguiu) [reply README.md:26]: Promscale has two components, the Promscale Connector and the Promscale Database. The Promscale Database is PostgreSQL, TimescaleDB AND the Promscale extension.

Do you think that is confusing?

File: README.md [hunk 4]
```diff
@@ -13,61 +13,118 @@
...
-[PromQL Compliance Test Suite](https://promlabs.com/promql-compliance-tests/).
-
-For OpenTelemetry, Jaeger and Zipkin users, Promscale provides the ability to run deep analysis
-on traces using SQL while visualizing the distributed traces in Jaeger or Grafana.
-Install our lightweight [opentelemetry-demo](https://github.com/timescale/opentelemetry-demo)
-on your computer to quickly get a feel of what's possible. More details in
-[this blog post](https://www.timescale.com/blog/learn-opentelemetry-tracing-with-this-lightweight-microservices-demo/).
+<img alt="Promscale" src="docs/assets/promscale-logo.png" width="600px">
+
+Promscale is a unified metric and trace observability backend for Prometheus,
```
[the remark this answers is not in the document]
Comment (ramonguiu) [reply README.md:18]: I've added key features under the use cases. I kept it short so it doesn't become very long. I think features are more relevant in the context of the problem you want to solve but open to put them in the introduction if you all feel that's better.

File: README.md [hunk 5]
```diff
@@ -13,61 +13,118 @@
...
+[Application Performance Management (APM) experience](https://docs.timescale.com/promscale/latest/visualize-data/apm-experience/)
+in Grafana built using SQL queries on traces.
 
-For a detailed description of the initial architecture of Promscale for Prometheus metrics that covers
-some key design principles, please see [our design doc](https://tsdb.co/prom-design-doc).
+* **Trace analysis**<br/>
+Jaeger searching capabilities are limited to filtering individual traces. This is helpful when troubleshooting problems once you know
+what you are looking for. With Promscale you can use SQL to interrogate your trace data in any way you want and discover issues
+that would usally take you a long time to figure out by just looking at log lines, metric charts or individual traces. You can see some
+examples in [this blog post](https://www.timescale.com/blog/what-are-traces-and-how-sql-yes-sql-and-opentelemetry-can-help-us-get-more-value-out-of-traces-to-build-better-software/)
```
Comment (alejandrodnm) [inline README.md:86]: Could we put a simple example here? At least just the SQL explaining what we are querying. I think this is a key point of the power of the product, and having to read an entire blogpost might not be very friendly.

Maybe something like:

```
SELECT
    client_span.service_name AS client_service,
    server_span.service_name AS server_service,
    server_span.span_name AS server_operation,
    count(*) AS number_of_requests,
    ROUND(sum(server_span.duration_ms)::numeric) AS total_exec_time
FROM
    span AS server_span
    JOIN span AS client_span
    ON server_span.parent_span_id = client_span.span_id
WHERE
    client_span.start_time > NOW() - INTERVAL '30 minutes' AND
    client_span.service_name != server_span.service_name
GROUP BY
    client_span.service_name,
    server_span.service_name,
    server_span.span_name
ORDER BY
    server_service,
    server_operation,
    number_of_requests DESC;
```
    
> In this query we are leveraging another powerful capability of SQL: joins. We are joining the span table with itself to identify only the spans that represent a call between two services. 

I think even though it might not make total sense on a first glance it might give an idea on the power of using SQL. If we could show (in a short paragraph) how cumbersome it would be to get the same information without SQL would be plus.

File: README.md [hunk 6]
```diff
@@ -13,61 +13,118 @@
...
 <img src="docs/assets/promscale-arch.png" alt="Promscale Architecture Diagram" width="800"/>
 
-This repository contains the source code of the Promscale Connector. Promscale also requires that the Promscale Extension
-which lives in [this repository](https://github.com/timescale/promscale_extension) is installed in the TimescaleDB/PostgreSQL
-database. The extension sets up and manages the database schemas and provides performance and SQL query experience improvements.
+## Promscale for Prometheus
+
+Promscale solves two main Prometheus use cases:
+
+* **Efficient long-term trend analysis and capacity planning**<br/>
```
[the remark this answers is not in the document]
Comment (ramonguiu) [reply README.md:37]: Thanks. Capacity planning is essentially analysis of trends, now? (how fast CPU or memory consumption is growing for example).

If you think they are completely independent we can break it up or maybe rephrase.

File: README.md [hunk 7]
```diff
@@ -13,61 +13,118 @@
...
+## Promscale Repositories
+
+This repository contains the source code of the Promscale Connector. Promscale also requires that the Promscale extension
+which lives in [this repository](https://github.com/timescale/promscale_extension) is installed in the TimescaleDB/PostgreSQL
+database. The extension sets up and manages the database schemas and provides performance and SQL query experience improvements.
+
 This repository also contains the source code for **prom-migrator**. **Prom-migrator** is
 an **open-source**, **community-driven** and **free-to-use**, **universal prometheus
-data migration tool**, that migrates data from one storage system to another, leveraging Prometheus's
+data migration tool**, that migrates datag from one storage system to another, leveraging Prometheus's
```
Comment (alejandrodnm) [inline README.md:120]:
```suggestion
data migration tool**, that migrates data from one storage system to another, leveraging Prometheus's
```

File: README.md [hunk 8]
```diff
@@ -13,61 +13,118 @@
...
-some key design principles, please see [our design doc](https://tsdb.co/prom-design-doc).
+* **Trace analysis**<br/>
+Jaeger searching capabilities are limited to filtering individual traces. This is helpful when troubleshooting problems once you know
+what you are looking for. With Promscale you can use SQL to interrogate your trace data in any way you want and discover issues
+that would usally take you a long time to figure out by just looking at log lines, metric charts or individual traces. You can see some
+examples in [this blog post](https://www.timescale.com/blog/what-are-traces-and-how-sql-yes-sql-and-opentelemetry-can-help-us-get-more-value-out-of-traces-to-build-better-software/)
+
+**Key features:** native OTLP support, SQL queries, APM capabilities, data compression, data retention
+
+**Try it out** by installing our lightweight [opentelemetry-demo](https://github.com/timescale/opentelemetry-demo) with a single
```
Comment (alejandrodnm) [inline README.md:90]: Do we have a demo for prometheus?

File: README.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (VineethReddy02) [reply README.md:37]: *Capacity planning* doesn't click a button for me. Feels like losing the context.

File: README.md (hunk 8 above)
Comment (VineethReddy02) [reply README.md:90]: We do not have one, Usually, if users deploy Prometheus it already has its own metrics as a bonus they can also scrape metrics from Promscale Connector to ingest some metric data into Promscale.

Comment (ramonguiu) [reply README.md:90]: We should have one though. The general getting started that Vineeth is putting together could be a good starting point and if we see many people use it then we can look into offering a more specific and detailed one.

File: README.md [hunk 9]
```diff
@@ -13,61 +13,118 @@
...
+
+* **Single-pane-of-glass across all your Kubernetes clusters**<br/>
+Use Promscale as a centralized storage for all your Prometheus instances
+so you can easily query data across all of them in Grafana and centralize
+[alert management](https://docs.timescale.com/promscale/latest/alert/) and
+[recording rules](https://docs.timescale.com/promscale/latest/downsample-data/recording/).
+
+**Key features**: 100% PromQL-compliant, high availability, multi-tenancy, PromQL alerting and recording rules, per-metric retention.
+
+If you use Prometheus and are also familiar with PostgreSQL, then Promscale is a great choice for
```
[the remark this answers is not in the document]
Comment (ramonguiu) [reply README.md:51]: True and I think we need to talk to a specific target market instead of the entire Prometheus market. Casting a wide net is not working well for us anyway. If you are not familiar with Postgres, then yeah, you don't get much benefits from using our product. Most users that use us already pick us because of Postgres, why not make it even clearer?

This is like the timescaleDB positioning of Postgres for Timeseries. We are also narrowing who we are targeting and we do it on purpose.

File: README.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (ramonguiu) [reply README.md:37]: I've rewritten this section a bit and also moved single pane of glass to the top since that's likely the main reason users look for a product like ours and the value prop there is clearer and easier to understand.

File: README.md [hunk 10]
```diff
@@ -13,61 +13,118 @@
...
-on your computer to quickly get a feel of what's possible. More details in
-[this blog post](https://www.timescale.com/blog/learn-opentelemetry-tracing-with-this-lightweight-microservices-demo/).
+<img alt="Promscale" src="docs/assets/promscale-logo.png" width="600px">
+
+Promscale is a unified metric and trace observability backend for Prometheus,
+Jaeger and OpenTelemetry built on PostgreSQL and TimescaleDB.
+
+Promscale serves as a robust and 100% PromQL-compliant Prometheus remote storage
+and as a durable and scalable Jaeger storage backend.
+
```
[the remark this answers is not in the document]
Comment (ramonguiu) [reply README.md:23]: I was trying to keep this short but I'll add something

File: README.md (hunk 5 above)
Comment (ramonguiu) [reply README.md:86]: I am not sure the readme of the repo is the place to put this. I wanted to put folks to our documentation for querying traces which also has examples but we it's still being reviewed.

Comment (alejandrodnm) [reply README.md:94]: Sounds good, I think I just wanted to give people reading this a glimpse of what they can achieve by using SQL to query the telemetry data.

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (ramonguiu) [reply README.md]: I've added key features at the top now

**Review by alejandrodnm — APPROVED** [review]

**Review by VineethReddy02 — APPROVED** [review]

**Review by Harkishen-Singh — APPROVED** [review]

**Comment by ramonguiu** [comment]
I'm having issue rebasing the PR so I've created a new one: https://github.com/timescale/promscale/pull/1483
